### ISSUE TITLE
Geographically similar not displaying for ADS archives

### ISSUE THREAD

**Issue by hew503** [codarchlab/ariadne-portal#184]
When viewing items from the ADS 'archives' metadata, geographically similar isn't displaying. I think this was the first metadata tested in the portal, so it may be that things were adjusted later? Here are some examples. It doesn't seem to be related to resource type.

http://portal.ariadne-infrastructure.eu/page/19138450
http://portal.ariadne-infrastructure.eu/page/19138292

cheers

Holly

**Comment by borsna** [comment]
These resources does not contain any coordinates:
http://portal.ariadne-infrastructure.eu/page/19138450 -> http://portal.ariadne-infrastructure.eu/data/19138450
http://portal.ariadne-infrastructure.eu/page/19138292 -> http://portal.ariadne-infrastructure.eu/data/19138292 

This is required to get the map displayed on the resource landing page.
We should check if the XML provided to MORe contains this information

**Comment by eafiontzi** [comment]
As you can see here http://more.locloud.eu:8080/objects/4467/19138450/ADS_ARCHIVE/content
the native resource does not contain coordinates, just bounding boxes

**Comment by borsna** [comment]
Thanks Eleni!
To get this working we may need to implement both single coordinates and bounding boxes in the geo structure (and search queries) for the portal.
This could require some reworking

**Comment by hew503** [comment]
Thanks for all the responses! :)

Rather than using the bounding box coordinates, we had also discussed converting the TGN to coordinates. So for the Star Carr example:

World, Europe, United Kingdom, England, North Yorkshire, Star Carr [7032612]

We also discussed converting the OSGB coordinates to WGS84, which I thought was what was decided, but I guess it wasn't implemented. 

Again for Star Carr, this is what it looks like in the metadata provided:

<country type="OSGB Grid Ref">
<easting>502700</easting>
<northing>481000</northing>
</country>

Either instance should allow a single set of coordinates to be derived, and while bounding boxes are often more accurate when it comes to describing the resource, we also provide coordinates for exactly this reason.

I think we will be happy with whatever is the best solution, but there are several options in the metadata. I just don't think we realised it hadn't been resolved and that was why they weren't displaying.

Many thanks!

Holly

**Comment by eafiontzi** [comment]
Hello again,
We have enriched the package with a service that receives as input a place name and returns as output a set of coordinates. So many of the resources now contain coordinates like http://portal.ariadne-infrastructure.eu/page/19138167 which did not have before. However, many placenames contain the code like the one mentinoned in your comment, and could not be mapped to coordinates

**Comment by hew503** [comment]
Hi Eleni

Is there a way to know approximately how many were successfully enriched and how many are still without coordinates? It will help us understand whether the problem is still big or small. Also, when you refer to the code I mentioned in my comment, I'm not sure if you mean the TGN code or the latitude/longitude code, as the metadata contains both, so I'm not sure which you have enriched.

Many thanks

Holly

**Comment by eafiontzi** [comment]
Hello,
There have been enriched 66 out of the 770 items in the package. We are using the geonames database http://www.geonames.org/ for the enrichment. The placenames which are trying to be matched are the TGN of the element location, for example "World, Europe, United Kingdom, Scotland, Dumfries and Galloway [7008074]", which brings no matches.
Best

**Comment by jfihn** [comment]
Calculate middle of bounding box and add as point in elasticsearch for research to be included.

**Comment by eafiontzi** [comment]
Hello all,

We have developed a new enrichment service that calculates the middle point of a bounding box and adds it as a coordinate to each resource. We have enriched a package from ADS, specifically the one including the examples sent above.

While in the item http://portal.ariadne-infrastructure.eu/page/19138292 the center is appearing fine, in the item http://portal.ariadne-infrastructure.eu/page/19138450 the center is a coordinate somehow lost at sea. The explanation is that the bounding box is very wide and expands to almost the whole world. How should we proceed with that?
